tgui/packages/tgui/interfaces/common/JobToIcon.ts

```diff
@@ -27,7 +29,11 @@ const BASEICONS = {
   'Cyber Police': 'qrcode',
   Cyborg: 'robot',
   Detective: 'user-secret',
+  'Diet Wizard': 'hat-wizard',
+  'Discount Godzilla': 'ticket-alt' //like a movie ticket? IDK theres not many good icons for this.
```

Comment:
```suggestion
  'Discount Godzilla': 'ticket-alt', //like a movie ticket? IDK theres not many good icons for this.
```

Reply:
Yeah... I need to find a different icon for godzilla and skeleton currently as they are borked

Reply:
Should be good now.